src/blocks.js

```diff
@@ -3092,6 +3092,17 @@ BlockMorph.prototype.userMenu = function () {
         }
         return menu;
     }
+    if (contains(
+        ['doBroadcast', 'doBroadcastAndWait', 'receiveMessage'],
```

Comment:
Also, we should probably check for fancy senders of '__shout__go__' and '__clone__init__', right? So we might want to include the green-flag and "When-I-start-as-clone" hats also.